core/Core.rdf

```diff
@@ -2391,7 +2666,7 @@ NL definition: OAGIS and CCO</iof-av:adaptedFrom>
 		<rdfs:label xml:lang="en-US">service provider role</rdfs:label>
 		<skos:example xml:lang="en-US">FedEx; Home-cleaning service; aircraft maintenance service; internet service provider</skos:example>
 		<iof-av:adaptedFrom>https://en.wikipedia.org/wiki/Service_provider</iof-av:adaptedFrom>
-		<iof-av:firstOrderLogicAxiom>ServiceProvider(x) → AgentRole(x) ∧ ∃y, ∃p ((Organization(y) ∨ Person(y)) ∧ (CommercialService(p)  ∨ (OfferingForSaleBusinessProcess(p)) ∧ ∃c(CommercialServiceAgreement(c) ∧ hasParticipantAtSomeTime(p,c)) ∧ participatesInAtSomeTime(y, p) ∧ roleOf(x, y))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>ServiceProvider(x) → AgentRole(x) ∧ ∃y, ∃p ((Organization(y) ∨ Person(y)) ∧ (CommercialService(p) ∨ (OfferingForSaleBusinessProcess(p) ∧ ∃c(CommercialServiceAgreement(c) ∧ hasParticipantAtSomeTime(p,c)))) ∧ participatesInAtSomeTime(y,p) ∧ roleOf(x,y))</iof-av:firstOrderLogicAxiom>
```

Comment:
"∃y, ∃p" does not comply with https://www.cs.ox.ac.uk/people/james.worrell/lecture9-2015.pdf as required by https://oagi.atlassian.net/wiki/spaces/IOF/pages/4532142081/IOF+Annotation+Property+Guide+V2.2.

I would suggest "∃y∃p" (or "∃y,p").